ste/xfer-anyToRemote.go

```diff
@@ -289,7 +290,23 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s ISenderBase, si
 		}
 	}
 
-	s.Epilogue()
+	s.Epilogue() // Perform service-specific cleanup before jptm cleanup. Some services may actually require setup to make the file actually appear.
+	
+	if info.S2SDestLengthValidation {
+		if s2sc, isS2SCopier := s.(s2sCopier); isS2SCopier { // TODO: Implement this for upload and download?
+			destLength, err := s2sc.GetDestinationLength()
+
+			if err != nil {
+				jptm.FailActiveSend("epilogueWithCleanupSendToRemote", err)
```

Comment:
Minor: I think the first parameter here is "when" and we normally would put something like "getting destination length" in there, so that if the error is ever reported, it can be narrowed down to the particular operation in question, rather than just to the method.